integrations/wagmi-v2/src/constants.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license Apache-2.0
+ */
+
+import type { Rdns } from "./types.js";
+
+export const SUPPORTED_RDNS: Rdns[] = [
+	"io.metamask",
+	"io.metamask.mobile",
+	"com.brave.wallet",
```

Comment:
Wrapping some providers might result in breaking the execution, when I was testing it was failing with OKX Wallet. Thats why the whitelist was added.

Reply:
Is incompatibility with OKX a known bug?

Reply:
![image](https://github.com/user-attachments/assets/9028ca14-1815-4c7d-8011-7e72ea7ef709)
Looks like the OKX wallet does not support Sapphire network.